### PR TITLE
KEYCLOAK-9670: add content to newsletter

### PR DESCRIPTION
**Pull request by sebastienblanc**

### CODE REVIEW

File: blog/keycloak-newsletter-1.ftl [hunk 1]
```diff
@@ -1,53 +1,53 @@
 <#--
 title = Keycloak Community Newsletter #1
-date = 2019-03-11
-publish = false
+date = 2019-03-26
+publish = true
```
Comment (stianst) [inline blog/keycloak-newsletter-1.ftl]: Change to publish = false and send a notification to Keycloak Team that the first newsletter is ready to be published asking if anyone has something to add. We can publish after giving folks a chance to propose additions

File: blog/keycloak-newsletter-1.ftl [hunk 2]
```diff
@@ -1,53 +1,53 @@
...
+Let's start with Philip Riecks who explains in <a href="https://rieckpil.de/howto-microprofile-jwt-authentication-with-keycloak-and-react/">this article</a> how you can leverage Microprofile JWT Authentication with Keycloak and React.
 </p>
-
-<h2>News from the Identity Management World</h2>
 <p>
-<ul>
-    <li>item one</li>
-    <li>item two</li>
-</ul>
+Ramandeep Singh has been <a href="https://medium.com/@ramandeep.singh.1983/enterprise-web-app-authentication-using-keycloak-and-node-js-c10b0e26b80d">blogging</a> about Keycloak and NodeJS and Joshua Alfred Erney explain in this <a href="https://www.jerney.io/secure-apis-kong-keycloak-1/">blog serie</a> how to integrate Keylcoak and <a href="https://konghq.com/">Kong</a>, a popular API management platform.
```
Comment (stianst) [inline blog/keycloak-newsletter-1.ftl]: You are referring to two different things in same sentence. Split into separate paragraphs or at least separate sentences.

File: blog/keycloak-newsletter-1.ftl [hunk 3]
```diff
@@ -1,53 +1,53 @@
...
 </p>
-
-<h2>Upcoming talks / Webinars</h2>
+<P>
+With Mohamed Aboullaite's blog post you will learn how to <a href="https://aboullaite.me/secure-kibana-keycloak/">secure your Kibana dashboards using Keycloak</a> and finally in <a href=https://beyondthekube.com/identity-management-for-on-prem-clusters/?utm_sq=g0u3m590zf">this 3 parts article</a> installing Keycloak on Kubernetes will have no more secrets for you.
```
Comment (stianst) [inline blog/keycloak-newsletter-1.ftl]: You are referring to two different things in same sentence. Split into separate paragraphs or at least separate sentences.

File: blog/keycloak-newsletter-1.ftl [hunk 4]
```diff
@@ -1,53 +1,53 @@
...
-    <li>item two</li>
-</ul>
+Ramandeep Singh has been <a href="https://medium.com/@ramandeep.singh.1983/enterprise-web-app-authentication-using-keycloak-and-node-js-c10b0e26b80d">blogging</a> about Keycloak and NodeJS and Joshua Alfred Erney explain in this <a href="https://www.jerney.io/secure-apis-kong-keycloak-1/">blog serie</a> how to integrate Keylcoak and <a href="https://konghq.com/">Kong</a>, a popular API management platform.
 </p>
-
-<h2>Upcoming talks / Webinars</h2>
+<P>
+With Mohamed Aboullaite's blog post you will learn how to <a href="https://aboullaite.me/secure-kibana-keycloak/">secure your Kibana dashboards using Keycloak</a> and finally in <a href=https://beyondthekube.com/identity-management-for-on-prem-clusters/?utm_sq=g0u3m590zf">this 3 parts article</a> installing Keycloak on Kubernetes will have no more secrets for you. 
+</p>  
+<h2>News from the project</h2>
```
Comment (stianst) [inline blog/keycloak-newsletter-1.ftl]: What's this section supposed to contain? Just relisting of other blog posts we've done or something else?

File: blog/keycloak-newsletter-1.ftl [hunk 5]
```diff
@@ -1,53 +1,53 @@
...
-date = 2019-03-11
-publish = false
+date = 2019-03-26
+publish = true
 author = Sébastien Blanc
 -->
 
 <p>
-intro
+This is the very first "Keycloak Community Newsletter". The goal of this newsletter is to share the news around the Keycloak project : contributions, technical articles, future plans of the team etc...
```
Comment (stianst) [inline blog/keycloak-newsletter-1.ftl]: Replace:
The goal of this newsletter is to share the news around the Keycloak project : contributions, technical articles, future plans of the team etc... 

With:

The goal of this newsletter is to share news around the Keycloak project.

File: blog/keycloak-newsletter-1.ftl [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
-    <li>item two</li>
-</ul>
+Keycloak 5.0.0 has been <a href="https://www.keycloak.org/2019/03/keycloak-500-released.html">released</a> and 6.0.0 is around the corner.
 </p>
-
-<h2>We want your contributions !</h2>
 <p>
+From now on, new larger Keycloak's features will be discussed in the open.For each new feature, a design document will be created and pushed to our Github repository as a simple MarkDown file. This will make it easy for everyone to comment as
+well as contribute to the designs by opening Github issues and providing pull requests. 
+We have already 3 documemts open for discussion :
```
Comment (stianst) [inline blog/keycloak-newsletter-1.ftl]: Missing space after . in "open.For each new feature"

Shouldn't have space before : in "discussion : "

File: blog/keycloak-newsletter-1.ftl (hunk 4 above)
Comment (sebastienblanc) [reply blog/keycloak-newsletter-1.ftl]: More like a recap of activities from the "core" team.

**Comment by sebastienblanc** [comment]
@stianst Yes, that would be awesome if you can add some items for the "the section highlights for contributions"

**Pull request merged**